src/app/services/http.service.ts

```diff
@@ -103,4 +104,22 @@ export class HttpService {
     public saveSkill(skill: Skill): Observable<any> {
         return this.httpClient.post<any>(environment.api + '/skills', {name: skill.name});
     }
+
+    public getLocations(): string[] {
+        const locations: Array<string> = [];
+        this.httpClient.get<Location[]>(environment.api + '/locations').subscribe(data => {
+                data.forEach(loc => {locations.push(loc.name); }); });
```

Comment:
Meerdere dingen sluiten achter elkaar maakt het niet overzichtelijk `}); });`. Zou deze of op aparte regels sluiten, met juiste inspringing. Of deze niet met curly braces openen aangezien er maar 1 ding plaats vindt in de subscribe en de foreach. Dan zou je er bijvoorbeeld dit van kunnen maken:

```typescript
this.httpClient.get<Location[]>(environment.api + '/locations').subscribe(data => data.forEach(loc => locations.push(loc.name)));
```

Reply:
Je zou met map/filter achtige constructie dit wellicht volledig naar 1 regel kunnen omvormen, maar dat gaat mij voor nu nog net iets te ver qua eigen kennis :)